app/src/main/res/values/color_palette.xml

```diff
@@ -67,4 +67,9 @@
   <color name="color_palette_checkbox_selected_state_color">@color/color_def_checkbox_green</color>
   <color name="color_palette_mark_completed_text_color">@color/color_def_white</color>
   <color name="color_palette_mark_completed_all_text_color">@color/color_def_white</color>
+  <color name="color_palette_hints_and_solution_summary_card_border_color">@color/color_def_accessible_light_grey_2</color>
```

Comment:
These colors shouldn't be specifically tied to hints and solutions, but instead tied to the types of UI elements being colorized.